external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -114,10 +114,6 @@ use move_package::{
 };
 use move_symbol_pool::Symbol;
 
-/// Enabling/disabling the language server reporting readiness to support go-to-def and
-/// go-to-references to the IDE.
-pub const DEFS_AND_REFS_SUPPORT: bool = true;
```

Comment:
This flag was only used very early on so that we had a kill switch for more advanced functionality. Not quite working anymore and not used anymore so killed it as part of this PR